### PR TITLE
move all deps to requirements, generalize Makefile for non-LLNL devs

### PR DESCRIPTION
**Pull request by ben-bay**
Addresses issue #21

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -89,10 +85,6 @@ install-tasks:
 	$(PIP) install -e .
```
[the remark this answers is not in the document]
Comment (ben-bay) [reply Makefile]: Agreed.

[the remark this answers is not in the document]
Comment (ben-bay) [reply Makefile]: Fixed.

File: Makefile [hunk 2]
```diff
@@ -59,26 +57,24 @@ PENV=merlin$(PYV)
...
-all: install install-tasks install-pip-mysql install-sphinx
+all: install-dev install-tasks install-workflow-deps install-pip-mysql
 
 
 # install requirements
-install: virtualenv
-	$(VENV)/bin/easy_install cryptography
-	$(PIP) install --cert $(CERT) -r requirements.txt
+install-dev: virtualenv
+	$(PIP) install -r requirements.txt
```
[the remark this answers is not in the document]
Comment (ben-bay) [reply Makefile]: Fixed.

File: docs/source/getting_started.rst [hunk 3]
```diff
@@ -1,7 +1,12 @@
...
 ================
 
-Quick Start Setup
+Quick Start
+++++++++++++++
+::
+    pip3 install merlinwf
+
+Developer Setup
 ++++++++++++++++++
```
[the remark this answers is not in the document]
Comment (ben-bay) [reply docs/source/getting_started.rst]: Addressed.

**Comment by ben-bay** [comment]
@lucpeterson anything else you'd like to see here?

File: docs/source/getting_started.rst [hunk 4]
```diff
@@ -5,53 +5,20 @@ Quick Start
...
 ::
     pip3 install merlinwf
+    
+All set up? See the :doc:`Merlin Commands <./merlin_commands>` section for using merlin.
+
 
 Developer Setup
 ++++++++++++++++++
+To install with the additional developer dependencies, use::
+    pip3 install merlinwf[dev]
```
Comment (lucpeterson) [inline docs/source/getting_started.rst]: I think some users may need quotes, ie `pip3 install "merlinwf[dev]"`

**Pull request merged**